src/OpenSage.Game/Logic/Object/GameObject.cs

```diff
@@ -23,12 +23,12 @@ public sealed class GameObject : DisposableBase
         internal static GameObject FromMapObject(
             MapObject mapObject,
             AssetStore assetStore,
-            GameObjectCollection parent,
-            in Vector3 position,
+            GameObjectCollection gameObjects,
```

Comment:
Why did you rename that?

Reply:
since 'parent' was very misleading in that context